.generator/templates/model_utils.mustache

```diff
@@ -632,50 +632,49 @@ def get_simple_class(input_value):
     return type(input_value)
 
 
-def check_allowed_values(allowed_values, input_variable_path, input_values):
+def check_allowed_values(allowed_values, input_variable, input_values):
     """Raises an exception if the input_values are not allowed
 
     Args:
-        allowed_values (dict): the allowed_values dict
-        input_variable_path (tuple): the path to the input variable
+        allowed_values (list): the allowed_values list
+        input_variable (str): the name of the input variable
         input_values (list/str/int/float/date/datetime): the values that we
             are checking to see if they are in allowed_values
     """
-    these_allowed_values = list(allowed_values[input_variable_path].values())
```

Comment:
Why do we remove this line?

Reply:
I think it makes more sense to keep the argument consistent. Note that we know have 2 different inputs, one that comes from Endpoint and the other from models, which don't have the same data structure.